packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -394,4 +565,56 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   );
 };
 
+type MultipleComboboxProps<T> = {
+  selectedItems?: SelectOption<T>[];
+  selectedItemsRenderer?: (options: SelectOption<T>[]) => JSX.Element;
+  tagRenderer: (item: SelectOption<T>) => JSX.Element;
+  mRef: React.ForwardedRef<HTMLDivElement>;
+};
+
+const MultipleSelectedContainer = <T,>({
+  selectedItems,
+  selectedItemsRenderer,
+  tagRenderer,
+  mRef,
+}: MultipleComboboxProps<T>) => {
+  const selectedItemsElements = React.useMemo(() => {
+    if (!selectedItems) {
+      return [];
+    }
+
+    return selectedItems.map((item) => tagRenderer(item));
+  }, [selectedItems, tagRenderer]);
+
+  return (
+    <>
+      {selectedItems && selectedItemsRenderer && (
+        <div
+          className='iui-select-tag-container'
```

Comment:
Why here you used div with tag container class and bellow you used SelectTagContainer component?

Reply:
moved everything under select tag container